Lib/test/test_inspect.py

```diff
@@ -527,6 +527,15 @@ def monkey(filename, module_globals=None):
     def test_getsource_on_code_object(self):
         self.assertSourceEqual(mod.eggs.__code__, 12, 18)
 
+class TestGettingSourceOfToplevelFrames(GetSourceBase):
+    fodderModule = mod
+
+    def test_range_toplevel_frame(self):
+        self.assertSourceEqual(mod.currentframe, 1, 82)
```

Comment:
The `82` here (and below) is fragile, it would need to be updated after almost any minor change to the "fodder module". I suggest using `None`, which is equivalent to an open-ended slice, e.g. `lines[1:]`.